src/style.css

```diff
@@ -1 +1,98 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  margin: 0;
+  padding: 0 8px;
+}
```

Comment:
Avoid adding padding to the `body` element, as it can cause unexpected layout issues. If you need to set a default horizontal spacing, consider using a container element inside the body.